types/account.go

```diff
@@ -192,6 +204,17 @@ func (bz ValAddress) Format(s fmt.State, verb rune) {
 	}
 }
 
+// Returns boolean for whether two ValAddresses are Equal
+func (bz ValAddress) Equals(bz2 ValAddress) bool {
+	return (bytes.Compare(bz.Bytes(), bz2.Bytes()) == 0)
+}
+
+// Returns boolean for whether an AccAddress is empty
+func (bz ValAddress) Empty() bool {
```

Comment:
vs nil?  What do we do for Empty() and Equals() if any are nil?

Reply:
Oh, forgot that they can be nil.  Fixed.